### PR TITLE
🌐 Add French definition for "dissociative identity disorder"

### PR DESCRIPTION
**Pull request by CharlesMangwa**
Hey!

This PR aims to add the French definition for **"dissociative identity disorder"**. 

A link has been added to the existing English definition (and vice versa).

Current French definitions (or submitted via another PR) have also been linked directly.

Thanks in advance for the proofreading! 🙏

### CODE REVIEW

**Comment by deammer** [comment]
@CharlesMangwa this looks great! Good to merge, IMO. 

Optional suggestion: TDI used to be called "double personnalité" and "personnalité multiple", before the better term was coined. That could be worth mentioning?

```
Le trouble dissociatif de l'identité (TDI) (anciennement connu sous le nom de "double personalité" ou "personalité multiple") est une maladie mentale...
```

**Review by deammer — APPROVED** [review]

**Comment by CharlesMangwa** [comment]
Thanks for the proofreading @deammer!

Your suggestion is a great idea, it provides more context without having this translation being completely different from the original: just added it!

**Comment by tatianamac** [comment]
Thank you both for the work on this! I definitely want to ensure we respect and incorporate the relevant aspects of the language it’s written in as much as possible. 

To me it’s more important that it makes sense in the French context than be literal to its English counterpart. Siblings, but not identical twins. 💙

File: 11ty/definitions/fr_FR/trouble-dissociatif-de-l-identite.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Trouble dissociatif de l'identité
```
Comment (tatianamac) [inline 11ty/definitions/fr_FR/trouble-dissociatif-de-l-identite.md]:
```suggestion
title: trouble dissociatif de l'identité
```

**Review by tatianamac — APPROVED** [review]
🚀

**Comment by CharlesMangwa** [comment]
@tatianamac Title is lowercase now and I've also added a definition reference that I used but forgot to mention in the description!

**Pull request merged**